wlauto/workloads/uxperf_applaunch/__init__.py

```diff
@@ -0,0 +1,168 @@
+#    Copyright 2015 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# pylint: disable=attribute-defined-outside-init
+import os
+
+from time import sleep
+
+from wlauto import Workload, AndroidBenchmark, AndroidUxPerfWorkload, UiAutomatorWorkload
+from wlauto import Parameter
+from wlauto import ExtensionLoader
+from wlauto import File
+from wlauto import settings
+from wlauto.exceptions import ConfigError
+from wlauto.utils.android import ApkInfo
+
+import wlauto.common.android.resources
+
+
+class UxperfApplaunch(AndroidUxPerfWorkload):
+    """
+    A workload that helps to launch other workloads and measure the application
+    launch time
+    """
```

Comment:
I shall add this as an improvement once I finish getting it run for all workloads.